tests/test_app.py

```diff
@@ -0,0 +1,33 @@
+"""Module for testing the server"""
+
+import os
+import tempfile
+import pytest
+from app import app
+
+
+class TestApp:
```

Comment:
TestAverageUndernourishment might be better, and we just have a different test file for each function.